traefik/traefik.toml

```diff
@@ -0,0 +1,23 @@
+defaultEntryPoints = ["http", "https"]
+
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+    [entryPoints.http.redirect]
+      entryPoint = "https"
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+
+[web]
```

Comment:
See https://www.digitalocean.com/community/tutorials/how-to-use-traefik-as-a-reverse-proxy-for-docker-containers-on-ubuntu-16-04?comment=77114  where I commented on what I realized works now getting Traefic up and running. Web is deprecated and so something like:
```
entrypoints]
  #...
  # https://docs.traefik.io/configuration/api/#authentication
  [entrypoints.webentry]
    address = ":8080"
    [entrypoints.webentry.auth]
      [entrypoints.webentry.auth.basic]
        users = ["admin:$apr1$HURTFrz.$Hn3i.LpEV1Cs34Z6dYZ.m/"]
```
should be used.